### PR TITLE
Refactor breaking news template to use new inline icon

### PR DESCRIPTION
**Pull request by phamann**
### Before

google chrome
### After

google chrome

/cc @sndrs @sturobson

### CODE REVIEW

**Comment by sammorrisdesign** [comment]
kitsonthumb

**Comment by paperboyo** [comment]
Hi,

Fixes https://github.com/guardian/frontend/issues/8655. All pagination icons still broken, though.

Regards
Mateusz

**Comment by sndrs** [comment]
is this round?

screen shot 2015-03-24 at 15 28 59

:+1:

File: static/src/javascripts/projects/common/modules/onward/breaking-news.js [hunk 1]
```diff
@@ -88,11 +89,13 @@ define([
                 if (alerts.length) {
                     $breakingNews = $breakingNews || bonzo(qwery('.js-breaking-news-placeholder'));
                     marque36icon = svgs('marque36icon');
+                    closeIcon = svgs('closeCentral');
```
Comment (sndrs) [inline static/src/javascripts/projects/common/modules/onward/breaking-news.js]: this ought to be `closeCentralIcon` really, so you know it comes with the `icon` class

Comment (sndrs) [inline static/src/javascripts/projects/common/modules/onward/breaking-news.js]: unless it doesn't of course...

**Comment by sammorrisdesign** [comment]
OMG It's not...

screen shot 2015-03-24 at 15 30 09

**Comment by sammorrisdesign** [comment]
kitsonthumbdown

**Comment by phamann** [comment]
The question is, was it ever round?

**Comment by phamann** [comment]
@sndrs @sammorrisdesign THIS IS NOW ROUND!

google chrome

**Comment by sammorrisdesign** [comment]
kitsonhappy

**Pull request merged**